.gitignore

```diff
@@ -1,4 +1,3 @@
 .env
 .DS_Store
-output_file.html
```

Comment:
so you might to return to ignoring this, and adjust the name of the other file, to show it is a test input file.
Maybe `sample_output.html`.